Makefile.am

```diff
@@ -23,6 +23,10 @@ EXTRA_DIST = conf_parser.y conf_tokenizer.l \
 	$(man1_MANS) $(man5_MANS)
 
 
+common.o: conf_parser.tab.o
```

Comment:
Aren't these supposed to be found by the dependency generation?

Reply:
*.tab.[ch] are generated, so I'd say no.

Reply:
Also, running make -j32 a few times shows that this is needed. Did you try ?

Reply:
I'm starting to worry that we're significantly increasing complexity for a not very common use case (given that there aren't 32 files to compile, `-j16` or `-j32` wouldn't make much difference). I know from experience that arbitrary projects are not safe to compile with `-j` unless you have done something like `make deps` beforehand. I don't mind shipping the dependencies in the `.tar.gz`, but the compiler should be able to generate all of those for us (whether or not the `.c` file is hand created or not), and it's much safer to have it do so, rather than encoding some by hand.

Reply:
Well, it FTBFS. So even 2.2 can't be accepted into Debian for that as is, I have to provide a patch to fix this dependency problem.